agents/pfeiffer_tpg366/pfeiffer_tpg366_agent.py

```diff
@@ -1,113 +1,117 @@
        close closes the socket
-    
+
     Todo:
         -Generalize IP address inputs (input IP  via command line?)
     """
-    def __init__(self, ip_address=IP_ADDRESS, port=8000, timeout=10, f_sample = 2.5):
+    def __init__(self, ip_address, port, timeout=10,
+                 f_sample=2.5):
         self.ip_address = ip_address
         self.port = port
         self.comm = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         self.comm.connect((self.ip_address, self.port))
         self.comm.settimeout(timeout)
-    
+
     def read_pressure(self, ch_no):
         """
         Function to measure the pressure of one given channel
         ch_no is the chanel to be measured (e.g. 1-6)
         returns the measured pressure as a float
-        
+
         Args:
             ch_no: The channel to be measured (1-6)
+
         Returns:
             pressure as a float
         """
-        msg = 'PR%d\r\n'%ch_no
+        msg = 'PR%d\r\n' % ch_no
         self.comm.send(msg.encode())
-        status = self.comm.recv(BUFF_SIZE).decode()#Could probably use this to catch exemptions
+        # Can use this to catch exemptions, for troubleshooting
+        status = self.comm.recv(BUFF_SIZE).decode()
         self.comm.send(ENQ.encode())
         read_str = self.comm.recv(BUFF_SIZE).decode()
         pressure_str = read_str.split(',')[-1].split('\r')[0]
         pressure = float(pressure_str)
-        #print(pressure)
-        return pressure        
+        return pressure
 
     def read_pressure_all(self):
         """measure the pressure of all channel
         Return an array of 6 pressure values as a float array
 
         Args:
             None
+
         Returns:
-            6 element array corresponding to each channels pressure reading, as floats
+            6 element array corresponding to each channels
+            pressure reading, as floats
         """
         msg = 'PRX\r\n'
         self.comm.send(msg.encode())
-        status = self.comm.recv(BUFF_SIZE).decode()#Could probably use this to catch exemptions
+        # Could use this to catch exemptions, for troubleshooting
+        status = self.comm.recv(BUFF_SIZE).decode()
         self.comm.send(ENQ.encode())
         read_str = self.comm.recv(BUFF_SIZE).decode()
         pressure_str = read_str.split('\r')[0]
         gauge_states = pressure_str.split(',')[::2]
-        gauge_states = np.array(gauge_states, dtype=int)#this is recorded just incase
+        gauge_states = np.array(gauge_states, dtype=int)
         pressures = pressure_str.split(',')[1::2]
         pressures = [float(p) for p in pressures]
-        #print(pressures,type(pressures[0]),"This works")
-        return pressures 
+        return pressures
 
     def close(self):
-        """
-        Close the socket of the connection
-        """
+        """Close the socket of the connection"""
         self.comm.close()
 
+
 class pfeifferAgent:
```

Comment:
Same as the pfeiffer class, please format to `PfeifferAgent`.

https://www.python.org/dev/peps/pep-0008/#class-names